### PR TITLE
Shubham dalal work

### PR DESCRIPTION
**Pull request by shubhamdalal49**

### CODE REVIEW

File: Shubham_Dalal/HTML/ASSIGNMENT5/shubham_format_page.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
```
Comment (nileshp1122) [inline Shubham_Dalal/HTML/ASSIGNMENT5/shubham_format_page.html]: check indentation
extra space in your code

File: Shubham_Dalal/HTML/ASSIGNMENT6/Shubham_Ass6_ques1.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
```
Comment (nileshp1122) [inline Shubham_Dalal/HTML/ASSIGNMENT6/Shubham_Ass6_ques1.html]: check indentation
extra spce in your code

**Review by nileshp1122 — CHANGES_REQUESTED** [review]

File: Shubham_Dalal/HTML/ASSIGNMENT5/shubham_format_page.html (hunk 1 above)
Comment (Ashwin2812) [inline Shubham_Dalal/HTML/ASSIGNMENT5/shubham_format_page.html]: check indentation
remove Extra Spaces

**Review by Ashwin2812 — CHANGES_REQUESTED** [review]

**Review by Ashwin2812 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

File: Shubham_Dalal/HTML/ASSIGNMENT6/Shubham_Ass6_ques1.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        <div class="heading1">
+            <h2>MI-11 Intern HTML Assessment</h2>
+        </div>
+        <hr />
+        <div class="heading2">
+            <h3><i>Personal Details</i></h3>
+        </div>
+        <form class="form1">
+            <label>First Name: &nbsp;</label>
+            <input type="text" name="name" placeholder="First Name" /><br /><br />
```
Comment (SnehaMakade) [inline Shubham_Dalal/HTML/ASSIGNMENT6/Shubham_Ass6_ques1.html]: placeholders second words 1st letter should be small not capital like First name, so make this change in all input boxes.

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do changes as per comments given

**Review by SnehaMakade — APPROVED** [review]

**Pull request merged**